net-misc/gallery-dl/gallery-dl-1.15.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python3_{7,8})
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit eutils distutils-r1 optfeature
+
+DESCRIPTION="Download image galleries and collections from several image hosting sites"
+HOMEPAGE="https://github.com/mikf/gallery-dl"
+
+if [[ "${PV}" == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mikf/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/mikf/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+RESTRICT="test"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="bash-completion zsh-completion"
+
+RDEPEND="
+	>=dev-python/requests-2.11.0[${PYTHON_USEDEP}]
+"
+
+# tests require network access
+#distutils_enable_tests setup.py
```

Comment:
Why is it commented out? RESTRICT="test" disables tests already

Reply:
Ok, I'll remove it. I left it there to point which tests the pkg used.